config/cache.ts

```diff
@@ -0,0 +1,19 @@
+export const CACHE_DEFAULT_HEADERS =
```

Comment:
looks like it should be in this repo (https://github.com/lidofinance/warehouse/tree/main/packages/next/api-wrapper)

Reply:
actually, not. This can be configurable, but if this will be moved to warehouse, it will be "hidden variable", it is better to be explicit

Reply:
used @lidofinance/next-cache-files-middleware